web-registry/src/pages/Projects/hooks/useProjectsSellOrders.tsx

```diff
@@ -50,46 +52,42 @@ const getProjectDisplayData = async (
   limit: number,
 ): Promise<ProjectWithOrderData[]> => {
   const projectsWithOrderData = await Promise.all(
-    projects
-      .sort(sortProjectsBySellOrdersAvailability(sellOrders))
```

Comment:
Removing this line is the only change here. We had two sorts doing the same thing.

Reply:
@mhagel Actually this line was useful because, for the homepage, we need to have features projects that sell orders first. It works like this:
Fetch projects + sell orders -> sort projects that have sell orders first -> take only first X projects if limit param is given -> make additional queries (ie: metadata) on remaining projects.

This ensures that we have projects with sell orders first and at the same time, we're limiting the number of queries needed.
The second sort you're mentioning would come too late because it would sort on a subset of projects already.